spec/queries/search_results_query_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SearchResultsQuery, type: :model do
+  describe '#call' do
+    context 'when given NO filters' do
```

Comment:
Should we use one of them `when`/`given`? We usually use `given:
```suggestion
    context 'given NO filters' do
```